langchain/src/callbacks/handlers/tracer_langchain.ts

```diff
@@ -48,24 +30,19 @@ export class LangChainTracer
     "Content-Type": "application/json",
   };
 
-  sessionName: string;
-
-  sessionExtra?: LangChainTracerFields["sessionExtra"];
-
-  protected session: TracerSession;
+  sessionName?: string;
 
   exampleId?: string;
 
-  tenantId?: string;
-
   caller: AsyncCaller;
 
+  timeout = 5000;
```

Comment:
Is this default too long? I suppose it's not a problem in TS since it's async but 5s * 3 seems long